### DISCUSSION TITLE
How to make a disk image with a volume of more than 4 GB?

### DISCUSSION THREAD

**Discussion by Piligrim-77** [erichelgeson/BlueSCSI-v1#232] [Q&A]
Hi. Please tell me how to create a disk image larger than 4 GB on a Windows computer? I don't have macOS or Linux. 
I found the Disk Jockey program, but it creates disk images for a volume of only up to 4 GB. 
Link to the program:
[(https://diskjockey.onegeekarmy.eu/)](https://diskjockey.onegeekarmy.eu/)
I will be glad if someone describes a way to create disk images in the format ".hda" or will share a ready-made image for 8 GB or 16 GB. 
Thank you in advance.

**Comment by erichelgeson** [comment]
On windows 10/11 there is a command called `fsutil` that can create an empty file of arbitrary size:
`fsutil file createNew HD0-4gb.hda 4294967296` for 4gb. The size argument is in bytes so you'll have to calculate it out.

DiskJockey has a community form here - i think he'll be releasing a version soon to create very large volumes https://github.com/OneGeekArmy/Disk-Jockey-Community/issues

There is no format to an hda file (well until your vintage machine formats it) so any file will do. heck you could even create a 4gb text file and it would work :)

Hope that helps you

**Reply by erichelgeson** [reply]
DJ windows now supports up to 256gb image file creation.

**Reply by Piligrim-77** [reply]
That's great news. Thank you for your help.
